openshift/templates/db/db-build.yaml

```diff
@@ -10,7 +10,6 @@ objects:
       labels:
         name: ${NAME}
         app: ${APP_NAME}${SUFFIX}
-        app.kubernetes.io/part-of: ${APP_NAME}${SUFFIX}
```

Comment:
Kube app labels were stripped: if we're applying them to all builds we should re-add them.